### ISSUE TITLE
Come up with visual way to represent map uncertainties

### ISSUE THREAD

**Issue by jarcangeli** [multidworlds/eigenspectra#35]

**Comment by jarcangeli** [comment]
Yep, probably lots of ways to go about this. I was thinking we could do something with transparency to de-emphasise regions that are poorly constrained, not sure how that would look.

**Comment by rodluger** [comment]
The degeneracies/covariances between pixels/spherical harmonic coefficients make it really hard to come up with an economical visualization of the uncertainties on an actual image of the map. My suggestion would be to show the maximum likelihood map and then a bunch (10, 20, 30?) of independent posterior draws to convey the spread in the possible solutions...
